gen/flights/ai_flight_planner_db.py

```diff
@@ -130,6 +131,7 @@
 
 CAP_CAPABLE = [
     Su_57,
+    F_16I,
```

Comment:
This is better at CAP than the F-22? I don't buy that.

Reply:
I put it where it belongs